layouts/partials/article_metadata.html

```diff
@@ -4,13 +4,25 @@
 
   {{ if $.Params.authors }}
   {{ $authorLen := len $.Params.authors }}
-  <div itemprop="author">
+  {{ if eq $authorLen 0}}
```

Comment:
This block is likely redundant and can be removed - the range statement should cover the single author case.

Reply:
This block is for the case when authors list is empty. I have changed it to `span`, `meta` tags.